Fix:

error: address argument to atomic operation must be a pointer to non-const _Atomic type ('const _Atomic(bool) *' invalid)

https://github.com/php/php-src/issues/8881
https://github.com/php/php-src/pull/11931
--- a/Zend/zend_atomic.h.orig	2024-02-13 09:22:59.000000000 -0600
+++ b/Zend/zend_atomic.h	2024-03-03 17:14:09.000000000 -0600
@@ -23,7 +23,7 @@
 	((__GNUC__ == (x) && __GNUC_MINOR__ >= (y)) || (__GNUC__ > (x)))
 
 /* Builtins are used to avoid library linkage */
-#if __has_feature(c_atomic) && defined(__clang__)
+#if __has_feature(c_atomic) && defined(__clang__) && defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201710L
 #define	HAVE_C11_ATOMICS 1
 #elif ZEND_GCC_PREREQ(4, 7)
 #define	HAVE_GNUC_ATOMICS 1
